nautobot/docs/development/index.md

```diff
@@ -42,7 +42,7 @@ The following provides more detail on these.
 * The GitHub **Release Milestone** will track items for the _current_ release.
 
 !!! note
-    Release windows and dates will be updated per [Release Management](release-management).
+    Release windows and dates will be updated per [Release Management](#release-management).
```

Comment:
This is literally linking to the parent section of this note. :-) I'm actually wondering if we'd do better to just remove this note, as it's really not clear to me what it's even saying.

Reply:
I know... and unfortunately this has been broken since we forked from Netbox so I have no idea what this really should point to. I will just remove the note.